torchvision/datasets/cifar.py

```diff
@@ -83,6 +82,8 @@ def __init__(self, root, train=True,
             self.train_data = np.concatenate(self.train_data)
             self.train_data = self.train_data.reshape((50000, 3, 32, 32))
             self.train_data = self.train_data.transpose((0, 2, 3, 1))  # convert to HWC
+
+            self.train_data = [Image.fromarray(self.train_data[i]) for i in range(self.train_data.shape[0])]
```

Comment:
I believe this can be simplified as follows
```python
self.train_data = [Image.fromarray(img) for img in self.train_data]
```
Also the line is probably too long and lint is complaining